libcst/_nodes/expression.py

```diff
@@ -517,8 +517,23 @@ class BaseString(BaseExpression, ABC):
 
 
 class _BasePrefixedString(BaseString, ABC):
-    @abstractmethod
-    def _get_prefix(self) -> str:
+    @property
+    def prefix(self) -> str:
+        """
+        Returns the string's prefix, if any exists.
+
+        See `String and Bytes literals
+        <https://docs.python.org/3.7/reference/lexical_analysis.html#string-and-bytes-literals>`_
+        for more information.
+        """
+        ...
+
+    @property
+    def quote(self) -> str:
```

Comment:
Can we use Literal to be specific?
```suggestion
    def quote(self) -> Literal['"', "'", ...]:
```

Reply:
I think that will fix typing errors when sending the result back to a new string/f-string construction too.